documentation/modules/managing/proc-drain-cleaner-certs.adoc

```diff
@@ -0,0 +1,154 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-drain-cleaner.adoc
+
+[id='proc-drain-cleaner-certs-{context}']
+= Adding or renewing the TLS certificates used by the Strimzi Drain Cleaner
+
+[role="_abstract"]
+The Drain Cleaner uses a webhook to receive eviction notifications from the Kubernetes API.
+The webhook uses a secure TLS connection and authenticates using TLS certificates.
+If you are not using `cert-manager` to deploy the Drain Cleaner, you need to generate the TLS certificates. 
```

Comment:
Here and in the note on line 18 => we should probably cover both cert-manager as well as the _OpenShift certificate injector_ here as both of them handle the renewal them self. This procedure basically applies only if the users generated the certificate on their own.

Reply:
Okay. I've updated to include OpenShift handling.